src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/DevicePortalConnection.cs

```diff
@@ -0,0 +1,310 @@
+using System;
+using System.Linq;
```

Comment:
nit: please add the following header to this file:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.